docs/infrastructure/deployment-targets/cloud-target-discovery/index.md

```diff
@@ -95,4 +115,45 @@ By configuring a well-known variable and tagging your Azure Web App appropriatel
 }]
 ```
 
-Octopus will now discover the web app as a target before deploying to it, matching the environment, role, and project from the deployment to the tags created with the ARM template, without any custom scripts or manual registration! Octopus will also remove this target if it is later removed from Azure in future releases of this feature.
+Octopus will now discover the web app as a target before deploying to it, matching the environment, role, and project from the deployment to the tags created with the ARM template, without any custom scripts or manual registration! Octopus will also remove this target if it is later removed from Azure.
+
+### Amazon ECS
+
+Let's say you have an project in Octopus called _Pet Shop_ that deploys an application to an Amazon ECS Cluster in a _Development_ environment using a role of _PetShopFrontEnd_ and the cluster is dynamically created as part of the deployment using a CloudFormation template.
+
+To use this cluster previously in Octopus you might have either registered the target manually, or used a [script step](/docs/infrastructure/deployment-targets/dynamic-infrastructure/new-octopustarget.md) with custom code to try and find and create the cluster target. In addition, previously when this cluster was no longer needed you might have needed to either [run a script](/docs/infrastructure/deployment-targets/dynamic-infrastructure/remove-octopustarget.md) or manually remove the target in Octopus.
+
+By configuring some well-known variables and tagging your ECS cluster appropriately, Octopus can discover this target for you at deployment time. Additionally, Octopus will continue to monitor the target, and will remove it if it is removed in AWS.
+
+- Set the credentials to use during discovery by either
```

Comment:
There are a few options here (worker creds vs account, IAM role or not) which makes it a little hard to digest down to a single bullet point. If there are suggestions on a better way to organise this that would be great.

Reply:
This is the most succinct way to organize it. If you saw merit in it, the only other approach I can think of would be to use a further set of sub-headings for each type of credentials. FWIW I think that would probably be too much for an examples section unless you wanted to add in things like screenshots.